pkg/controller/topology.go

```diff
@@ -200,6 +211,8 @@ func GenerateAccessibilityRequirements(
 		hash, index := getPVCNameHashAndIndexOffset(pvcName)
 		i := (hash + index) % uint32(len(requisiteTerms))
 		preferredTerms = sortAndShift(requisiteTerms, nil, i)
+	} else if strictTopology {
+		preferredTerms = requisiteTerms
```

Comment:
This entire "else" close is for the "Delayed binding" case, right? Better move the comment up or add a separate comment, because now it's at first glance not obvious which case is handled here.

Reply:
yes, I agree. I made the suggested change.